serverless/functions/reference-content/code-examples.mdx

```diff
@@ -37,7 +37,7 @@ There are several ways to return a response from a handler function:
 **Response object with HTTP information:**
 
 ```python
-def my_handler(event, context):
+def handle(event, context):
```

Comment:
I like this change a lot because it makes it possible to cc/cv directly in the console :+1: